debs/SPECS/3.9.0/wazuh-agent/debian/rules

```diff
@@ -22,6 +22,7 @@ export TARGET_DIR=${CURDIR}/debian/wazuh-agent
 export INSTALLATION_DIR="/var/ossec"
 export INSTALLATION_SCRIPTS_DIR="${INSTALLATION_DIR}/packages_files/agent_installation_scripts"
 export JOBS="5"
```

Comment:
I read in README that, by default, this will be 4. Please change it to 2, let's be conservatives.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/139/commits/53094e480ac245621ccb8ba1f3944d09cdb3163e.